src/components/Layout/AppLayout.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+import { JustChildrenProp } from '@/types';
+import { ViewPort } from '@styles/theme/components/Misc';
+
+export const AppLayout: React.FC<JustChildrenProp> = ({ children }) => {
+  return <ViewPort bg="white">{children}</ViewPort>;
+};
```

Comment:
```suggestion
export const AppLayout: React.FC<JustChildrenProp> = ({ children }) => (
  <ViewPort bg="white">{children}</ViewPort>
)
```

Reply:
I should definitely add this rule to the `.eslintrc`

Reply:
This case I fixed in this PR ([f20aad9](https://github.com/dbrosio3/blackmarket-frontend/pull/14/commits/f20aad9969d466e129a2b627deac53604d413171))

The rest of the unnecessary return statements I removed here: #16